ExampleMod/Content/NPCs/ExampleBoneMerchant.cs

```diff
@@ -24,6 +25,15 @@ namespace ExampleMod.Content.NPCs
 	public class ExampleBoneMerchant : ModNPC
 	{
 		private static Profiles.StackedNPCProfile NPCProfile;
+		private static Asset<Texture2D> shimmerGun;
+
+		public override void Load() {
+			shimmerGun = ModContent.Request<Texture2D>(Texture + "_Shimmer_Gun");
+		}
+
+		public override void Unload() {
+			shimmerGun = null;
+		}
```

Comment:
Not necessary, I know we don't unload ALCs properly yet, but I don't want to mislead modders into thinking they have to clean up every static field they ever make